### ISSUE TITLE
SwiftUI based previews based on real screenshots and different text settings

### ISSUE THREAD

**Issue by mirkokiefer** [blackbeltlabs/TextAnnotation#20]
Let's define a file in Xcode with ten different text renderings based on real world screenshots and different font settings.
This way we can quickly iterate on ideal results.

Ideally, we could even automate saving those views via a CI tool (Travis or move to Github Actions - whichever is easier) for previewing those on Github.

**Comment by ulian-onua** [comment]
@mirkokiefer where should we take these screenshots from? Should we paste several real-world screenshots images to the bundle?

About automation need to R&D if it is possible.

**Comment by mirkokiefer** [comment]
@ulian-onua yes let's just add some to the bundle - not packaged with the release though. Just pick a few for now - I can help replace with other ones later.
Also let's not focus on the automation piece now - if we can see the SwiftUI previews that should be fast enough to play with the parameters.

**Comment by ulian-onua** [comment]
@mirkokiefer  got it. I'll prepare them with a possibility to paste different attributes ([NSAttributedString.Key) that should allow us to play with different default settings.

**Comment by ulian-onua** [comment]
@mirkokiefer please check this branch `update/solid_outline` with the SwiftUI previews done.
The current SwiftUI previews are based on shadow and outline (strokeWidth and strokeColor) properties customization.

Please check `TextContainerView_Preview` file to view the previews. I added some comments there for clarification.

You can change `static var defaultColor: NSColor?` to set text color for all previews. This will be applied to all previews if the color is not overridden by a certain preview. You can use `TextColor.swift` to get all current colors that Zappy app uses. `TextColor` is convertible to `NSColor` and vise versa. 

You can change  `static var defaultImageName: String` to change the background image of previews. Currently `zapier_screenshot` and `zapier_screenshot2` is available.

If these settings are changes you will need to press "Resume" button on preview canvas
![image](https://user-images.githubusercontent.com/15073398/92374571-97fa0a00-f108-11ea-83e2-49a4988da8dd.png)

The settings of views inside the preview view can be changed on fly
